components/pages/Signin/ThirdPartySignInButtons.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @desc This module provides a component that renders the third-party sign-in buttons.
+ * @param {function} handleSignIn - The function that handles the sign-in process.
+ * @returns {JSX} The third-party sign-in buttons.
+ */
+import React from "react";
+import Icon from "../../application/widgets/Icon";
+// import styles from "../../../styles/scss/signin.module.scss";
+
+const ThirdPartySignInButtons = ({ handleSignIn, styles }) => {
+  return (
+    <div>
+      <div className={styles.dividerWithText}>
+        <span>OR</span>
+      </div>
+      <div className={styles.google}>
+        <a
+          className={styles.google}
+          style={{ backgroundColor: "#55acee" }}
+          onClick={() => handleSignIn("google")}
+          role="button"
+        >
```

Comment:
If it is a button, you can use \<button\>. If it has to be \<a\>, it should have a href attribute, which you may set the value as 'javascript:void(0)'.

Reply:
Done